python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -423,6 +456,47 @@ def quadraticCorrelations(self, xCorrList, fluxList, name):
 
         return meanXcorr
 
+    def sampleCovModel(self, mu, n, g, covModelList, flux, name):
```

Comment:
Would you mind adding a unit test or unit tests under the `tests` directory for this function?  (where the BF tests are). These tests are in addition to tests with actual data that this new code runs and produces expected results.